src/Widgets/ConnectionDialog.vala

```diff
@@ -138,6 +138,7 @@ public class Sequeler.Widgets.ConnectionDialog : Gtk.Dialog {
         color_picker = new Gtk.ColorButton.with_rgba ({ 222, 222, 222, 255 });
         color_picker.get_style_context ().add_class ("color-picker");
         color_picker.can_focus = false;
+        color_picker.set_tooltip_text(_("Select connection color"));
```

Comment:
Can you add a little space to make the linter happy?
```suggestion
        color_picker.set_tooltip_text (_("Select connection color"));
```

Reply:
Sure!